Controller/CoreController.php

```diff
@@ -129,10 +124,22 @@ public function searchAction(Request $request)
         }
 
         return $this->render($this->container->get('sonata.admin.pool')->getTemplate('search'), array(
-            'base_template' => $this->getBaseTemplate($request),
+            'base_template' => $this->getBaseTemplate(),
             'admin_pool'    => $this->container->get('sonata.admin.pool'),
             'query'         => $request->get('q'),
             'groups'        => $this->getAdminPool()->getDashboardGroups(),
         ));
     }
+
+    /**
+     * @return Request
+     */
+    private function getRequest()
```

Comment:
with symfony 2.x, this conflicts with the public method in the base controller (see travis-ci).

i suggest we call it getRequestObject here and add a note in phpdoc that this is only to be BC with symfony 2 and 3 and deprecated (even though its private).